@kiva/kv-components/vue/KvTextInput.vue

```diff
@@ -45,6 +45,16 @@
 				:icon="mdiAlertCircleOutline"
 				class="tw-absolute tw-top-1.5 tw-right-1.5 tw-pointer-events-none tw-text-danger"
 			/>
+			<button
+				v-if="canClear && valid"
+				type="button"
+				@click="clearInput"
+			>
+				<kv-material-icon
+					:icon="mdiClose"
+					class="tw-absolute tw-top-1.5 tw-right-1.5"
```

Comment:
small thing, but I think these classes would be better on the `<button>` el

Reply:
Done